### PR TITLE
Update gitignore

### PR DESCRIPTION
**Pull request by jamesachamp**
`bestbook.egg-info` directory is now ignored.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 venv/
+*.egg-info/
```
Comment (cclauss) [inline .gitignore:2]: Please put one and only one `\n`at the end of the file so the GitHub editor removes that ⛔

**Review by cclauss — APPROVED** [review]
https://github.com/github/gitignore/blob/master/Python.gitignore is what I usually use but this change LGTM

**Comment by jamesachamp** [comment]
Thanks!  Just bookmarked that repo for future use.

**Comment by cclauss** [comment]
When you create a new repo, the form will ask you if you want a `.gitignore` file and if you say yes, it will copy in this file.

**Pull request merged**